### PR TITLE
MINV-10: Add Svelte to Starter

### PR DESCRIPTION
**Pull request by jsam07**
# Description
This PR adds Svelte to the electron starter.

Fixes [MINV-10](https://runik.atlassian.net/browse/MINV-10)

## Major Changes Made
- Add [Svelte ](https://svelte.dev/)
- Add [Rollup](https://rollupjs.org/) as bundler
- Change app builder to [electron-builder](https://www.electron.build/index.html)

## How Has This Been Tested?
- Verified that the app runs locally, via:
  ```bash
  yarn electron-dev
  ```
  **Application Running in Dev Mode**
  image

- Verified that the application builds on Windows, via:
  ```bash
  yarn make
  ```
  **Packaged Artifact Running**
  image

## Checklist
- [x] Local Build

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: latest
 
       - name: Install node dependencies
-        run: npm i
```
Comment (jsam07) [inline .github/workflows/build.yml:23]: Some issues with `npm` resolving the dependency tree - let's track as a separate issue to dig into later.

File: package.json [hunk 2]
```diff
@@ -26,14 +30,21 @@
   },
   "homepage": "https://github.com/MINERVA-MD/minerva-core#readme",
   "devDependencies": {
-    "@electron-forge/cli": "^6.1.1",
```
Comment (jsam07) [inline package.json:29]: Had a build issue w/ `electron-forge` so opted to swap it out for `electron-builder`,  just temporarily. Can re-assess what builder we want - quite easy to swap out.

File: main.js [hunk 3]
```diff
@@ -1,31 +1,51 @@
...
-app.on('window-all-closed', () => {
-  if (process.platform !== 'darwin') {
-    app.quit()
-  }
-})
\ No newline at end of file
+    
+    // Open the DevTools and also disable Electron Security Warning.
+    // process.env['ELECTRON_DISABLE_SECURITY_WARNINGS'] = true;
+    // mainWindow.webContents.openDevTools();
```
Comment (matteomiceli) [inline main.js:32]: Are these comments left over from testing?

File: package.json (hunk 2 above)
Comment (matteomiceli) [reply package.json:29]: I've heard good things about electron-builder's optimization so I'm cool with it 👍

File: .github/workflows/build.yml (hunk 1 above)
Comment (matteomiceli) [reply .github/workflows/build.yml:23]: I'm cool with using yarn.

File: renderer.js
Comment (matteomiceli) [inline renderer.js:1]: Is `renderer` not required by electron? How are we connecting the svelte bundle to be served by electron?

File: src/main.js
Comment (matteomiceli) [inline src/main.js:1]: We looking to add TS support in the near future?

**Review by matteomiceli — APPROVED** [review]
Nice work 🚀 
Had a couple questions, but they're not blockers.

File: main.js (hunk 3 above)
Comment (jsam07) [reply main.js:32]: Left it as a reminder to set this up properly during `dev`. I can gut it and track it properly on Jira?

Comment (matteomiceli) [reply main.js:32]: Naw I'm not too worried so long as it's a helpful comment 👍

File: src/main.js
Comment (jsam07) [reply src/main.js:1]: Yep, tracking in [MINV-14](https://runik.atlassian.net/browse/MINV-14)

Comment (matteomiceli) [reply src/main.js:1]: Ahh nice, somehow missed that 🙈

File: renderer.js
Comment (jsam07) [reply renderer.js:1]: It's required, yep, but the name doesn't need to be `renderer .js`. In this case `main.js` loads `index.html` via:

```js 
mainWindow.loadURL('http://localhost:8080/');
```
`index.html` then loads the Svelte bundle via: 
```js 
<script defer src='/build/bundle.js'></script>
```

**Pull request merged**